src/coreclr/runtime.proj

```diff
@@ -31,8 +31,7 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <_CoreClrBuildScript Condition="$([MSBuild]::IsOsPlatform(Windows))">build-runtime.cmd</_CoreClrBuildScript>
-      <_CoreClrBuildScript Condition="!$([MSBuild]::IsOsPlatform(Windows))">build-runtime.sh</_CoreClrBuildScript>
+      <_CoreClrBuildScript>$([MSBuild]::NormalizePath('$(RepoRoot)', 'build-runtime$(ScriptExt)'))</_CoreClrBuildScript>
```

Comment:
@sandreenko Can you please check this condition? Am not quite sure is this correct or not!